packages/hardhat/package.json

```diff
@@ -7,6 +7,7 @@
     "check-types": "tsc --noEmit --incremental",
     "compile": "hardhat compile",
     "deploy": "hardhat deploy",
+    "deploy:canvas": "hardhat run deploy/canvasOnchain.ts --network sepolia",
```

Comment:
do we really need another script? Isn't the `deploy` script deploying everything that is on the `deploy` folder?

Reply:
Will make changes to that it's when i'm having issue with the network but forget to remove it later on